### PR TITLE
Add USD Trading account

### PR DESCRIPTION
**Pull request by zubair-deriv**
### Card
https://app.clickup.com/t/20696747/WALL-4244

### CODE REVIEW

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

File: cypress/support/commands/wallets.js [hunk 1]
```diff
@@ -121,6 +122,30 @@ Cypress.Commands.add('c_setupTradeAccountResponsive', (wallet) => {
...
+      cy.wrap(list)
+        .find('.wallets-card')
+        .first()
+        .find('.wallets-text')
+        .then(($el) => {
+          const text = $el.text().trim() // Trim any extra whitespace
+          cy.log('Text found:', text) // Log the actual text to debug
+          expect(text).to.include('USD')
+        })
+    })
```
Comment (afsaneh-rezaeinejad-deriv) [inline cypress/support/commands/wallets.js:149]: @zubair-deriv can you modify this command to switch to USD wallet if its needed, also better to make is generic

**Review by afsaneh-rezaeinejad-deriv — CHANGES_REQUESTED** [review]

File: cypress/support/commands/wallets.js [hunk 2]
```diff
@@ -121,6 +122,38 @@ Cypress.Commands.add('c_setupTradeAccountResponsive', (wallet) => {
...
+      cy.wrap(list)
+        .find('.wallets-card')
+        .first()
+        .find('.wallets-text')
+        .then(($el) => {
+          const text = $el.text().trim() // Trim any extra whitespace
+          cy.log('Text found:', text) // Log the actual text to debug
+          expect(text).to.include(wallet)
+          cy.findAllByText('Transfer').should('be.visible')
+          cy.wait(300)
```
Comment (abuzar-deriv) [inline cypress/support/commands/wallets.js:155]: Why this wait?

File: cypress/support/commands/wallets.js [hunk 3]
```diff
@@ -121,6 +122,38 @@ Cypress.Commands.add('c_setupTradeAccountResponsive', (wallet) => {
...
+      if (!firstChild.hasClass('wallets-progress-bar-active')) {
+        cy.get('.wallets-progress-bar .wallets-progress-bar-inactive')
+          .first()
+          .click()
+      }
+    })
+  cy.get('.wallets-progress-bar-active')
+    .parentsUntil('.wallets-carousel-content__cards')
+    .then((list) => {
+      cy.log('inside then')
```
Comment (abuzar-deriv) [inline cypress/support/commands/wallets.js:145]: Is this log needed? Looks more like something needed for debugging only.

File: cypress/e2e/smoke/Wallets/addTradingAccount.cy.js [hunk 4]
```diff
@@ -11,12 +11,26 @@ describe('QATEST-156146 - Add trading account to wallet account (Crypto)', () =>
...
+    () => {
+      cy.c_visitResponsive('/', 'small')
+      cy.findAllByText(/Wallet/, { timeout: 10000 }).should('exist')
+      cy.findAllByText(/Deposit/, { timeout: 10000 }).should('exist')
+      cy.c_skipPasskeysV2()
+      cy.findByText("Trader's Hub").should('be.visible')
+      cy.c_setupUSDTradeAccountResponsive('USD')
+    }
+  )
+
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/addTradingAccount.cy.js:33]: Can you please add testid for this?

**Review by abuzar-deriv — COMMENTED** [review]
Some questions..

File: cypress/support/commands/wallets.js (hunk 2 above)
Comment (zubair-deriv) [reply cypress/support/commands/wallets.js:155]: In responsive view, the carousel switched to USD so quickly, that even assertion on dom element is not enough to setup a trading account due to which I have to add this wait

File: cypress/support/commands/wallets.js (hunk 3 above)
Comment (zubair-deriv) [reply cypress/support/commands/wallets.js:145]: correct. It was added while debugging. 
I have removed this log.

File: cypress/e2e/smoke/Wallets/addTradingAccount.cy.js (hunk 4 above)
Comment (zubair-deriv) [reply cypress/e2e/smoke/Wallets/addTradingAccount.cy.js]: Updated

**Comment by zubair-deriv** [comment]
I am closing this PR, as this would not be a valid use case after migration.

**Pull request closed without merging**